docs/VM_SETUP.md

```diff
@@ -54,33 +56,46 @@
       [Source](https://stackoverflow.com/a/6448326/1753041)
    6. `sudo chmod -R g+rwX /home/ibf-user/IBF-system`
       [Source](https://stackoverflow.com/a/6448326/1753041)
-   7. Setup Environment Variables
+   7. Create build folder
+      - `mkdir /home/ibf-user/IBF-system/interfaces/IBF-dashboard/www`
+      - `chown azureuser:ibf-users /home/ibf-user/IBF-system/interfaces/IBF-dashboard/www`
+      - `chmod 775 /home/ibf-user/IBF-system/interfaces/IBF-dashboard/www`
+   8. Setup Environment Variables
       1. Create `/home/ibf-user/IBF-system/.env`
          1. `cp /home/ibf-user/IBF-system/example.env /home/ibf-user/IBF-system/.env`
          2. Set the appropriate values in the `.env` file
          3. Load the `.env` vars by `source /home/ibf-user/IBF-system/.env`
          4. Test if the vars were loaded correctly `echo $NODE_ENV`
-   8. Load certificate: load `DigiCertGlobalRootCA.crt.pem` in `services/API-service/cert` for connection to Azure Postgres server (if applicable)
-   9. `. tools/deploy.sh`
+   9. Load certificate: load `DigiCertGlobalRootCA.crt.pem` in `services/API-service/cert` for connection to Azure Postgres server (if applicable)
+   10. Configure Nginx
+       - `cp /etc/nginx/conf.d/default.conf /etc/nginx/conf.d/default.conf.backup`
+       - `cp tools/nginx.conf /etc/nginx/conf.d/default.conf`
+       - Change `root` and `proxy_pass` values in `/etc/nginx/conf.d/default.conf`
```

Comment:
and 'server_name' value?

Reply:
You're right. I'll update the docs to include `server_name`.